File: solutions/clojure/raindrops/1/src/raindrops.clj
(ns raindrops)

(defn convert [n]
  (let [ans
  (str (if (= (mod n 3) 0) "Pling" nil)
       (if (= (mod n 5) 0) "Plang" nil)
       (if (= (mod n 7) 0) "Plong" nil))]
    (if (empty? ans) (str n) ans))
)

File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"

namespace raindrops {

  std::string convert(int n) {
    std::string res{""};
    if (n % 3 == 0) {
      res += "Pling";
    }
    if (n % 5 == 0) {
      res += "Plang";
    }
    if (n % 7 == 0) {
      res += "Plong";
    }
    if (res == "") {
      return std::to_string(n);
    } else {
      return res;
    }
  }

}  // namespace raindrops
